app/controllers/bookings_controller.rb

```diff
@@ -10,16 +9,17 @@ def new
 
   def create
     @booking = Booking.new(booking_params)
+    @donut = Donut.find(params[:donut_id])
     @booking.user = @current_user
-    # get id off the donut from the params
-    # get instance of dognut
-    # @booking.donut =
-    @booking.save
-    redirect_to booking_path(@booking)
+    if @booking.save
+      redirect_to booking_path(@booking)
+    else
+      render :new, status: :unprocessable_entity
+    end
+    redirect_to donut_path(@donut)
```

Comment:
whats up with this second redirect here?

Reply:
it was just chilling ;)